modules/cnv-labels-nodes-workloadscheduling.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_advanced_vm_management/cnv-dedicated-resources-vm.adoc
+// * cnv/cnv_vm_templates/cnv-dedicated-resources-vm-template.adoc
+
+[id="cnv-labels-nodes-workloadscheduling_{context}"]
+
+= Preparing to enable dedicated resources
+
+Prepare to enable dedicated resources by doing the following.
```

Comment:
Recommend replacing this line with the one flagged in the concept module.

Reply:
Done. Thanks, much better. Sorry for this.